packages/core/src/app/checkout/CheckoutApp.tsx

```diff
@@ -67,4 +68,8 @@ export default class CheckoutApp extends Component<CheckoutAppProps> {
     private createStepTracker: () => StepTracker = () => {
         return createStepTracker(this.checkoutService);
     };
+
+    private createBodlService: (subscriber: (state: CheckoutSelectors) => void) => BodlService = (subscriber) => {
+        return createBodlService(subscriber);
```

Comment:
You should be able to do this here
```suggestion
        return createBodlService(this.checkoutService.subscribe);
```